### ISSUE TITLE
INPUT requires an accessible name (label)

### ISSUE THREAD

**Issue by thiessenp-cds** [cds-snc/platform-forms-client#1353]
A LABEL is missing on the title INPUT. Also placeholders are generally recommended against for people with cognitive disabilities. Recommend a visible LABEL.

Reference: [WCAG 2.4.6](https://w3c.github.io/wcag/21/guidelines/#headings-and-labels)
Reference: [WCAG 3.3.2](https://w3c.github.io/wcag/21/guidelines/#labels-or-instructions)

![Screenshot 2022-11-29 at 1.56.38 PM.png](https://images.zenhubusercontent.com/63321fcd2e391c72611f2316/f913d1a2-5028-4c36-b662-42a66db2f6c1)

![Screenshot 2022-11-29 at 2 07 25 PM](https://user-images.githubusercontent.com/107579368/204623153-6993c4b5-8b3f-41bb-a73d-0942825cff61.png)

**Comment by timarney** [comment]
Form title was updated in https://github.com/cds-snc/platform-forms-client/pull/1347

<img width="985" alt="Screen Shot 2022-11-30 at 11 50 15 AM" src="https://user-images.githubusercontent.com/62242/204858752-dfc3f0f6-4118-485e-83c3-f616fd87755b.png">

**Comment by timarney** [comment]
Question input have
<img width="941" alt="Screen Shot 2022-11-30 at 11 52 51 AM" src="https://user-images.githubusercontent.com/62242/204859262-9684631a-9893-4d09-b4ee-e2af5f59da26.png">

**Comment by timarney** [comment]
@thiessenp-cds let us know if we're good here or if a larger discussion needs to happen around placeholders --- which would be a larger UI change.

From my testing our usage of placeholder is correct in that the elements are still described properly.
